lib/crypto/algorithms/megolm.js

```diff
@@ -169,7 +169,11 @@ MegolmEncryption.prototype._prepareNewSession = function(room) {
  * @private
  *
  * @param {string} session_id
+ *
  * @param {Object<string, Object<string, boolean>|boolean>} shareMap
+ *    Map from userid to true (meaning this is a new user in the room, so all
+ *    of his devices need the keys), or a map from deviceid to true (meaning
+ *    this user has one or more new devices, which need the keys).
```

Comment:
this comment doesn't seem to match the typing of the param? what is the string->{string, boolean} scenario?